CHANGELOG.md

```diff
@@ -162,6 +162,7 @@ to allow empty coins in fp current rewards.
 - [#1485](https://github.com/babylonlabs-io/babylon/pull/1485) Add channel check on consumer FP creation
 - [#1516](https://github.com/babylonlabs-io/babylon/pull/1516) Remove `zoneconcierge` port ID from state
 - [#1492](https://github.com/babylonlabs-io/babylon/pull/1492) Bump cosmos evm to v0.3.1
+- [#1536](https://github.com/babylonlabs-io/babylon/pull/1536) Refactor `zoneconcierge` for optimize ibc channel, client io
```

Comment:
please, move changelog entry to the `Unreleased` list

Reply:
Updated,

btw, while merging #1540, I found an item in the linked discussion that may require further review, so I left [it there](https://github.com/babylonlabs-io/babylon/pull/1540#discussion_r2274815340). There might be a need for additional modifications, so I’d appreciate it if you could share your thoughts when you have time,  Thanks

Reply:
Hey @dongsam, yes, it is true what you're stating. However, the functions that are using the map, relay also on a second condition to be true to perform some operation (eg. have pending headers to broadcast, or staking events pending). If there's some kind of edition on the consumer registry or deactivation, this should impact these other functions.